IPython/core/magic.py

```diff
@@ -675,6 +676,9 @@ def __call__(self, *args, **kwargs):
                              "magic aliases cannot call themselves.")
         self._in_call = True
         try:
-            return fn(*args, **kwargs)
+            if self.magic_params:
```

Comment:
@takluyver, thanks for taking a look at my pull request!  Yea, that's the interpretation I had, but after thinking about it, I like your interpretation better.  It gives the user more flexibility and closer mimics the unix shell behavior.

I've updated the pull request to include the behavior you cited above which adds any called parameters after the alias parameters.  Can you take a look?